workshop-resources/keyboard-navigation/event-reviews/2022-10-12-notebook7.md

```diff
@@ -0,0 +1,210 @@
+# Notebook 7 - Collaborative keyboard navigation review
+
+Please add your findings as [GitHub suggestions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) so you are credited in the commit history!
+
+## Keyboard navigation accessibility review of [tool name]
+
+Reviewed via based on [this link we'll all be working off of]().
+
+- Review date:
+- Operating system(s):
+- Browser(s): 
+- Keyboard language(s) used:
+- Any other details (ie. dependencies, github repo, etc.):
+
+### Authors
+
+Participants may add their name below if they would like to be listed publicly as a part of the review.
+
+1.
+2.
+3.
+4.
+5. 
+6.
+7.
+8.
+9.
+10. 
+
+## Review
+
+### Content order
+
+When accessed via keyboard, the content order is logical. ([WCAG 2.2 Focus order](https://www.w3.org/TR/WCAG22/#focus-order))
+
+1. From the top of the page, press the Tab key repeatedly until reviewer reaches the end of content.
+2. Reviewers will know it is the end because if they press Tab once more their keyboard focus will return to the browser.
+3. If the reviewer cannot make it to the end of content, please take note of where the focus gets stuck.
+
+Does the content order follow reviewer expectations based on reading the content?
+Result:
+
+Does the content order make sense for interacting with the content?
+Result:
+
+Are any major content areas missed when navigating via keyboard?
+Result:
+
+Are any interactive content areas missed when navigating via keyboard?
+Result:
+
+Other notes or recommendations:
+
+### Areas to navigate
```

Comment:
Notes: this is best tested latest HEAD of Notebook which includes https://github.com/jupyter/notebook/pull/6487 so using if it works https://mybinder.org/v2/gh/jupyter/notebook/main?urlpath=tree